File: BackendAPI/src/main/scala/Entities/Match.scala
package Entities

import java.sql.Timestamp

import Controllers.KafkaControllers.CustomDeserializers.CustomerDateAndTimeDeserialize
import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.databind.annotation.{JsonDeserialize, JsonSerialize}

import scala.beans.BeanProperty

@JsonIgnoreProperties(ignoreUnknown = true)
@JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
case class Match(
                  @BeanProperty
                  id: Int,

                  @BeanProperty
                  homeTeamId: Int,

                  @BeanProperty
                  awayTeamId: Int,

                  @BeanProperty
                  managerId: Int,

                  @BeanProperty
                  locationId: Int,

                  @BeanProperty
                  sportId: Int,

                  @BeanProperty
                  tournamentId: Int,

                  @BeanProperty
                  scoreHomeTeam: Int,

                  @BeanProperty
                  scoreAwayTeam: Int,

                  @BeanProperty
                  IsStarted: Boolean,

                  @BeanProperty
                  date: String
                ) {
}

File: GenerateFilesWithData/GenerateFilesWithData/Match.cs
using System;

namespace GenerateFilesWithData
{
    public class Match
    {
        public int Id { get; set; }
        public int HomeTeamId { get; set; }
        public int AwayTeamId { get; set; }
        public int ManagerId { get; set; }
        public int LocationId { get; set; }
        public int SportId { get; set; }
        public int TournamentId { get; set; }
        public bool IsStarted { get; set; }
        public int ScoreHomeTeam { get; set; }
        public int ScoreAwayTeam { get; set; }
        public DateTime Date { get; set; }

        public Match(int id, int homeTeamId, int awayTeamId, int managerId, int locationId, int sportId,
            int tournamentId, bool isStarted, int ScoreHomeTeam, int ScoreAwayTeam, DateTime date)
        {
            this.Id = id;
            this.HomeTeamId = homeTeamId;
            this.AwayTeamId = awayTeamId;
            this.ManagerId = managerId;
            this.LocationId = locationId;
            this.SportId = sportId;
            this.TournamentId = tournamentId;
            this.IsStarted = IsStarted;
            this.ScoreHomeTeam = ScoreHomeTeam;
            this.ScoreAwayTeam = ScoreAwayTeam;
            this.Date = date;
        }
    }
}     